scripts/triggers-end-to-end-tests/tests.ts

```diff
@@ -172,6 +176,112 @@ describe("function triggers", () => {
     });
   });
 
+  describe("reproCase", () => {
```

Comment:
Hi @alexstroukov, help me out if you have some time to look at this - I tried setting up the repro and I'm getting the following errors: 

 1) function triggers
       reproCase
         "before all" hook for "should publish a message":
     Error: Timeout of 1000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/usr/local/google/home/christhompson/firebase/firebase-tools/scripts/triggers-end-to-end-tests/tests.ts)
      at listOnTimeout (node:internal/timers:564:17)
      at processTimers (node:internal/timers:507:7)

  2) function triggers
       reproCase
         "after all" hook for "should publish another message":
     Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
      at GoogleAuth.getApplicationDefaultAsync (node_modules/@google-cloud/pubsub/node_modules/google-auth-library/build/src/auth/googleauth.js:183:19)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at async GoogleAuth.getClient (node_modules/@google-cloud/pubsub/node_modules/google-auth-library/build/src/auth/googleauth.js:568:17)
      at async GrpcClient._getCredentials (node_modules/@google-cloud/pubsub/node_modules/google-gax/src/grpc.ts:202:20)
      at async GrpcClient.createStub (node_modules/@google-cloud/pubsub/node_modules/google-gax/src/grpc.ts:404:19)

Reply:
@christhompsongoogle I think perhaps it might be the way pubsub is instantiated. When pointing to a remote environment it wants proper security credentials like a keyfile but for our tests we simply give it a localhost api endpoint which points at the emulator

```
const pubsub = new PubSub({ apiEndpoint: 'localhost:8085' })
```